Nadim.CinemaReservationSystem.Web/ClientApp/src/Services/UserService.js

```diff
@@ -0,0 +1,69 @@
+import authenticationDataAccess from '../DataAccess/AuthenticationDataAccess';
```

Comment:
As AuthenticationDataAccess and UserService are strongly tied together it will be a good idea to give them the same name. I'm fine with both Authentication* and User* but you should choose only one.